Source/Core/DolphinQt/Settings/USBDeviceAddToWhitelistDialog.h

```diff
@@ -32,6 +32,8 @@ class USBDeviceAddToWhitelistDialog final : public QDialog
   static constexpr int DEVICE_REFRESH_INTERVAL_MS = 100;
   QTimer* m_refresh_devices_timer;
   QDialogButtonBox* m_whitelist_buttonbox;
+  QPushButton* add_button;
```

Comment:
@spycrab any reason why there seems to be a mix of `m_` and no `m_`? I don't remember if we made a specific exception to our naming conventions for Qt.

If there isn't any specific motivation, can we make these two new ones `m_`? (don't bother fixing the rest in the same PR).

Reply:
Didn't write this dialog myself (just rebased someone elses changes). 

I just didn't catch it, I guess.

Reply:
Thanks! @Zexaron can you rename these two new member variables to match our coding style then?